src/components/Layout/Header.tsx

```diff
@@ -414,7 +426,20 @@ class Header extends React.Component<HeaderProps, HeaderState> {
           </Box>
           <List className={classes.mobileMenuList}>
             {MenuItems.MainNavigation.map(item => {
-              const { mobileMenu, authRequired, to, label } = item
+              const { mobileMenu, authRequired, to, label, isPortalLink } = item
+              if (isPortalLink == true) {
```

Comment:
You have this in a couple spots - can you just check `isPortalLink` to be truthy or does it explicitly need to equal true?

Reply:
you're correct- looks like it doesn't need to be explicit. @erincdustin looks like you've got an update on your machine, cool if i just cherrypick and add it here?

Reply:
@esitarz yep just updated